partner_aging/unapplied_payments.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2014 Ursa Informative Systems (<www.ursainfosystems.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+###############################################################################
+#
+# Description:  Create sql views consisting of unreconciled customer/supplier
+#               deposits. Unapplied customer deposits are used in customer
+#               aging view
+#
+###############################################################################
+
+
+from openerp.osv import osv, fields
+from openerp import tools
+
+
+class customer_unapplied(osv.osv):
+
+    _name = 'account.voucher.customer.unapplied'
+    _auto = False
+
+    _columns = {
+        'partner_id': fields.many2one(
+            'res.partner',
+            u'Partner',
+            readonly=True
+        ),
+        'partner_name': fields.text('Name', readonly=True),
+        'salesman': fields.many2one('res.users', u'Sales Rep', readonly=True),
+        'avg_days_overdue': fields.integer(u'Avg Days Overdue', readonly=True),
+        'oldest_invoice_date': fields.date(u'Invoice Date', readonly=True),
+        'total': fields.float(u'Total', readonly=True),
+        'days_due_01to30': fields.float(u'01/30', readonly=True),
+        'days_due_31to60': fields.float(u'31/60', readonly=True),
+        'days_due_61to90': fields.float(u'61/90', readonly=True),
+        'days_due_91to120': fields.float(u'91/120', readonly=True),
+        'days_due_121togr': fields.float(u'+121', readonly=True),
+        'max_days_overdue': fields.integer(u'Days Overdue', readonly=True),
+        'current': fields.float(u'Total', readonly=True),
+        'invoice_ref': fields.char('Reference', size=128),
+        'invoice_id': fields.many2one(
+            'account.invoice',
+            'Invoice',
+            readonly=True
+        ),
+        'currency_name': fields.text('Currency', readonly=True),
+        'comment': fields.text('Notes', readonly=True),
+    }
+
+    _order = "partner_name"
+
+    def init(self, cr):
+
+        query = """
+SELECT cast(100000000000 + av.id AS BIGINT) AS id
```

Comment:
We can, but no one needs that as of yet, so I will leave that to whoever wants to do it.